java-manta-client/src/main/java/com/joyent/manta/client/multipart/EncryptedMultipartUpload.java

```diff
@@ -83,6 +89,19 @@ EncryptionState getEncryptionState() {
     }
 
     /**
```

Comment:
Give a _hint_ (not an explanation) the javadoc about why you are doing this and why it is package private.